plugins/MauticClearbitBundle/Form/Type/BatchLookupType.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
- * @copyright   2015 Mautic Contributors. All rights reserved
+ * @copyright   2019 Mautic Contributors. All rights reserved
```

Comment:
We should not change year in existing files. Only the new files should have the current year.
```suggestion
 * @copyright   2015 Mautic Contributors. All rights reserved
```